django/test/runner.py

```diff
@@ -434,7 +436,7 @@ def __init__(self, pattern=None, top_level=None, verbosity=1,
                  interactive=True, failfast=False, keepdb=False,
                  reverse=False, debug_mode=False, debug_sql=False, parallel=0,
                  tags=None, exclude_tags=None, test_name_patterns=None,
-                 pdb=False, buffer=False, **kwargs):
+                 pdb=False, buffer=False, no_faulthandler=False, **kwargs):
```

Comment:
I would suggest changing the code so you're able to use the simpler `faulthandler=True` as the argument name. This will also simplify the code below that reads `if not no_faulthandler`. (You can still keep `--no-faulthandler` as the exposed option though.)

Reply:
Agreed.

Reply:
I don't disagree.

Reply:
using `faulthandler=True` puts the `faulthandler` module out of scope. Should I change it to something else, or keep it as `no_faulthandler`?

Reply:
`no_faulthandler=False` -> `enable_faulthandler=True`